shared/src/test/diff/pretyper/ucs/SpecilizationCollision.mls

```diff
@@ -89,10 +90,11 @@ fun example3(t) =
 //│ | | | | | | | | | remaining cases should be covered by the wildcard
 //│ | | | | | | | | | checkCoverage <== TERM 42
 //│ | | | | | | | | checkCoverage ==> 0 diagnostics
-//│ | | | | | | | | remaining cases are not covered: []
+//│ | | | | | | | | remaining cases should be covered by the wildcard
+//│ | | | | | | | | checkCoverage <== TERM 42
 //│ | | | | | | | checkCoverage ==> 0 diagnostics
 //│ | | | | | | | Coverage checking result: 0 errors
-//│ fun example3: forall 'a. (Base & {#x: Num & 'a}) -> (Int | 'a)
+//│ fun example3: forall 'a. (Base & {#x: Num & 'a} | Object & ~#Base) -> (Int | 'a)
```

Comment:
I wonder why we don't see `Derived` and its field `y` in this inferred type. Seems like a bug.

Reply:
Any clue on this?

Reply:
Actually not a bug (see test we added in meeting).